src/components/MultiHopTrade/components/SharedTradeInput/SharedSettingsPopover.tsx

```diff
@@ -123,8 +123,15 @@ export const SharedSettingsPopover: FC<SharedSettingsPopoverProps> = memo(
       [handleSlippageTypeChange],
     )
 
-    const isHighSlippage = useMemo(() => bnOrZero(slippageAmount).gt(1), [slippageAmount])
-    const isLowSlippage = useMemo(() => bnOrZero(slippageAmount).lt(0.05), [slippageAmount])
+    const isHighSlippage = useMemo(
+      () => slippageAmount && bnOrZero(slippageAmount).gt(1),
+      [slippageAmount],
+    )
+    const isLowSlippage = useMemo(
+      () => slippageAmount && bnOrZero(slippageAmount).lt(0.05),
+      [slippageAmount],
+    )
+    console.log({ slippageAmount })
```

Comment:
ser

Reply:
Fixed in [9d54ef0](https://github.com/shapeshift/web/pull/9427/commits/9d54ef0b26c7b990957f2f4dce61741e50839a20)